service/raft/raft_group0.cc

```diff
@@ -1191,11 +1194,16 @@ collect_schema_versions_from_group0_members(
     static constexpr auto rpc_timeout = std::chrono::seconds{5};
     static constexpr auto max_concurrency = 10;
 
-    auto retry_period = std::chrono::seconds{1};
     std::unordered_map<gms::inet_address, table_schema_version> versions;
-    while (true) {
+    for (auto retry_period = std::chrono::seconds{1};
+        ;
+        co_await sleep_abortable(retry_period, as), retry_period = std::min(retry_period * 2, max_retry_period)) {
```

Comment:
See my other comments near `for (;;` above

Reply:
Changed to use sleep_with_exponential_backoff class.